rudder-core/src/main/scala/com/normation/rudder/services/reports/ExecutionBatch.scala

```diff
@@ -270,10 +272,12 @@ object ExecutionBatch extends Loggable {
    * Takes a string, that should contains a CFEngine var ( $(xxx) or ${xxx} )
    * replace the $(xxx) (or ${xxx}) part by .*
    * and doubles all the \
-   * Returns a string that is suitable for a being used as a regexp
+   * Returns a string that is suitable for a being used as a regexp, with anything not
+   * in ".*" quoted with \Q...\E.
+   * For example, "${foo}(bar)$(baz)foo" => ".*\Q(bar)\E.*\Qfoo\E"
```

Comment:
are you missing the first Q/E in your example ? I think it hsould be: "\Q\E ".*\Q(bar)\E.*\Qfoo\E"

Reply:
yes! Sorry